.github/supporting-files/ci/osl/export_vars.sh

```diff
@@ -79,8 +79,8 @@ if [[ -f "$ENV_FILE" ]]; then
 fi
 
 if [[ -n "${GITHUB_ENV:-}" ]]; then
-  # GitHub Actions: append into $GITHUB_ENV
-  cat "$MERGED" >> "$GITHUB_ENV"
+  # GitHub Actions: append into $GITHUB_ENV removing empty lines
+  grep -v -E '^\s*$' "$MERGED" >> "$GITHUB_ENV"
```

Comment:
Thi change prevents empty lines to cause the error:
```
[Build Midstream KIE Tools/build] invalid format '', expected a line with '=' or '<<'
```
This error happened to me locally using ACT, but I think this way the command is more robust on GH and allow us to use ACT locally